src/wallets/Wallets.tsx

```diff
@@ -1,55 +1,30 @@
-import React, {useEffect, useState} from 'react'
+import React, {useEffect} from 'react'
 import {pipe} from 'fp-ts/lib/function'
 import {option} from 'fp-ts'
 import {message} from 'antd'
 import {WalletState} from '../common/wallet-state'
 import {Navigate} from '../layout/Router'
 import {Loading} from '../common/Loading'
 import {Wallet} from './Wallet'
-import {rendererLog} from '../common/logger'
 import {useTranslation} from '../settings-state'
-import {useRecurringTimeout} from '../common/hook-utils'
 
 export const Wallets = (): JSX.Element => {
   const walletState = WalletState.useContainer()
 
-  useEffect(() => {
-    if (walletState.walletStatus === 'INITIAL') {
-      walletState.refreshSyncStatus()
-    }
-
-    rendererLog.debug(`walletStatus: ${walletState.walletStatus}`)
```

Comment:
It was pretty useful to have this as a debug log. Could you add it back somewhere?

Reply:
It logs every time when sync status is updated, not when walletStatus changed, so it doesn't help with tracking status changes.